virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -28,13 +28,17 @@ You can expedite VM creation by using templates that have an available boot sour
 
 Templates without a boot source are labeled *Boot source required*. You can use these templates if you complete the steps for xref:../../virt/vm_templates/virt-creating-vm-template.adoc#virt-adding-a-boot-source-web_virt-creating-vm-template[adding a boot source to the virtual machine].
 
-include::modules/virt-creating-vm-template.adoc[leveloffset=+2]
-
-[NOTE]
+[INFO]
```

Comment:
Hi @apinnick 
I was in the middle of the peer review process and hence I added the `peer-review-in-progress` label. But I noticed that you added the `peer-review-done` label and merged the PR.
Actually, I had one comment - The `[INFO]` block is not rendering properly in the preview. I did not find any guidance about `[INFO]` in the supplementary guide. IMHO `[NOTE]` would be better to keep. Can you please create a follow-up PR to update `[INFO]` --> `[NOTE]`? 
cc: @abhatt-rh 
<img width="857" alt="Screenshot 2023-08-07 at 4 29 11 PM" src="https://github.com/openshift/openshift-docs/assets/49194531/7c0f890b-1cfa-4f23-afef-5043ab26b28c">

Reply:
Sorry, I saw that your last comment was '/remove-label peer-review-needed', so I assumed that peer review was done. I will check the "info" admonition rendering and create a separate PR. Thanks.

Reply:
Looks like you are correct! I will change it back to "note"

Reply:
Thank you :)

Reply:
Thank-you for bringing it to my attention! I don't know why I used INFO instead of NOTE. I think someone suggested it and I assumed that it would work. Won't make that mistake again! :-)